app/services/users/profile.rb

```diff
@@ -70,6 +70,7 @@ def profile
         loa: user.loa,
         multifactor: user.multifactor,
         verified: user.loa3?,
+        sign_in: user.identity.sign_in,
```

Comment:
@saneshark @narin - Did you guys confirm that `user.identity` cannot return `nil` (i.e. favoring `sign_in: user.identity&.sign_in`)?

Reply:
Yes, if user.identity is nil you're not signed in, so users/profile won't be accessible -- authorization will fail. And sign_in hash has some pretty sane defaults in the event that the saml response has some nonsense in it, when I add account type i'll make sure the schema lists the possible string values for service_name and account_type.